c_api/index_io_c_ex.h

```diff
@@ -14,6 +14,7 @@
 
 #include <stdio.h>
 #include "IndexBinary_c.h"
+#include <faiss/replaceMalloc.h>
```

Comment:
Am I understanding this correctly that you importing the header file here implies that the APIs in only this method will take advantage of the malloc overrides and no where else?